tests/test_dataclasses.py

```diff
@@ -795,3 +799,34 @@ class Config:
     e.other = 'bulbi2'
     with pytest.raises(dataclasses.FrozenInstanceError):
         e.item.name = 'pika2'
+
+
+@dataclasses.dataclass
```

Comment:
can we move this and the model into the test function or do they need to be module level?

Reply:
If you need it to be module level, perhaps you could use `create_module`? see `test_forward_ref.py`